install_config/configuring_ephemeral.adoc

```diff
@@ -56,45 +58,44 @@ apiServerArguments:
    - LocalStorageCapacityIsolation=true
 ...
 ----
-```
-
-. Edit the ConfigMap for all nodes to enable the LocalStorageCapacityIsolation on the command line.  You can identify the ConfigMaps that need to be edited as follows:
 
-```
+. Edit the ConfigMap for all nodes to enable the LocalStorageCapacityIsolation on the command line. You can identify the ConfigMaps that need to be edited as follows:
++
+----
 $ oc get cm -n openshift-node
 NAME                  DATA      AGE
 node-config-compute   1         52m
 node-config-infra     1         52m
 node-config-master    1         52m
-```
-
-For each of these maps (node-config-compute, node-config-infra, and
-node-config-master), you need to add the featuregate:
-
-```
+----
++
+For each of these maps, `node-config-compute`, `node-config-infra`, and
+`node-config-master`, you need to add the feature gate:
++
+----
 oc edit cm node-config-master -n openshift-node
-```
-
-If there is already a feature-gates: declaration, you need to add
-
-```
+----
++
+If there is already a `feature-gates:` declaration, you need to add the following to the list of feature gates:
++
+----
 ,LocalStorageCapacityIsolation=true
-```
-
-to the list of feature gates.  If not, you need to add:
-
-```
+----
++
+If there is no `feature-gates:` declaration, you need to add:
```

Comment:
s/you need to add/add the following section under the \<section_this_goes_in>:
(I'd make this part of a ul in the third step.)